docs/assets/specification/ErgoSpec.Backend.ErgoBackend.jsx

```diff
@@ -0,0 +1,250 @@
+<div>
```

Comment:
How were these JSX files generated, please? Did you use the Docusarus migration scripts, for example?

Because these files are dynamically created, we should make sure that our scripts generate the correct output. This can wait until after this PR has been merged, but should be completed before merging to `master`

Reply:
According to Docusaurus v2 migration guide, they provided this website https://transform.tools/html-to-jsx for transforming html to jsx. So, I manually converted all html files to jsx.

Reply:
Thanks, @prakhars for your diligence here, however, unfortunately, these files are stale (they don't seem to have been updated in over 2 years).`

I believe that all of the files in `docs/assets/specification` should be deleted. @jeromesimeon can you confirm, please?